user/views.py

```diff
@@ -18,7 +17,7 @@ class CreateTokenView(ObtainAuthToken):
 
 class ManageUserView(generics.RetrieveUpdateAPIView):
     serializer_class = UserSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` line is commented out. If this is intentional, ensure that the default authentication settings are sufficient for your use case. Otherwise, consider uncommenting it to explicitly define the authentication method.